src/index.html

```diff
@@ -1,17 +1,168 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="#"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          class="input"
+          type="text"
+          id="surname">
+        <br>
+
+        <label for="name">Name:</label>
+        <input
+          class="input"
+          type="text"
+          id="name">
+        <br>
+
+        <label for="age">How old are You?</label>
+        <input class="input"
+          value="12"
+          type="number"
+          id="age"
+          min="12"
+          max="100">
```

Comment:
```suggestion
        <input class="input"
          value="12"
          type="number"
          id="age"
          min="12"
          max="100"
        >
```
Распологай везде закрывающую скобку на одной линии с открывающей